client/src/components/eventType/Calendar.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+    <div class="dates">
+        <div class="month">
+            <VBtn class="arrows prev-mth" text @click="onPrevMonth">&lt;</VBtn>
+            <div class="mth">{{ formattedDate }}</div>
+            <VBtn class="arrows next-mth" text @click="onNextMonth">&gt;</VBtn>
+        </div>
+        <div class="days-in-week">
+            <div v-for="day in weekDays" :key="day">{{ day }}</div>
+        </div>
+        <div class="days">
+            <div
+                v-for="index in firstMthDayInWeek"
+                :key="index + 'skipped'"
+            ></div>
+            <VBtn
+                :class="['day', isSelected(dayIndex) && 'white--text']"
+                :disabled="dayIndex < startEventDate || dayIndex > endEventDate"
+                :color="isSelected(dayIndex) ? '#281ac8' : 'rgb(243, 247, 247)'"
```

Comment:
for colors use pre-defined constants in `/plugins/vuetify.js`. #281ac8 is primary one